packages/admin/admin-rte/src/core/Rte.tsx

```diff
@@ -149,201 +154,232 @@ export const styleMap = {
     },
 };
 
-const Rte: React.RefForwardingComponent<any, RteProps & WithStyles<typeof styles>> = (props, ref) => {
-    const { value: editorState, onChange, options: passedOptions, disabled, classes } = props;
-    const editorRef = React.useRef<DraftJsEditor>(null);
-    const editorWrapperRef = React.useRef<HTMLDivElement>(null);
+export const Rte: React.ForwardRefExoticComponent<React.PropsWithoutRef<RteProps> & React.RefAttributes<unknown>> = React.forwardRef(
+    (inProps: RteProps, ref) => {
+        const {
+            value: editorState,
+            onChange,
+            options: rteOptions,
+            disabled,
+            minHeight,
+            colors,
+            slotProps,
+            ...restProps
+        } = useThemeProps({
+            props: inProps,
+            name: "CometAdminRte",
+        });
 
-    // merge default options with passed options
-    let options = passedOptions ? { ...defaultOptions, ...passedOptions } : defaultOptions;
+        const ownerState: OwnerState = {
+            disabled,
+            minHeight,
+        };
 
-    // extract deprecated options and handle them specially
-    let deprecatedCustomBlockMap: ICustomBlockTypeMap_Deprecated = {};
-    if (options.customBlockMap) {
-        deprecatedCustomBlockMap = options.customBlockMap;
-        delete options.customBlockMap;
-    }
+        const editorRef = React.useRef<DraftJsEditor>(null);
+        const editorWrapperRef = React.useRef<HTMLDivElement>(null);
 
-    cleanBlockTypeMap(options.blocktypeMap); // mutate object and print warning when configuration is wrong
+        // merge default options with passed options
+        let options = rteOptions ? { ...defaultOptions, ...rteOptions } : defaultOptions;
```

Comment:
Please keep the name `passedOptions`. Both default options as well as passed options are valid RTE options.